src/common/media.cpp

```diff
@@ -76,12 +77,134 @@ char getByte(GCodeFilter::State *state);
 static char gcode_buffer[MAX_CMD_SIZE + 1]; // + 1 for NULL char
 static GCodeFilter gcode_filter(&getByte, gcode_buffer, sizeof(gcode_buffer));
 static uint32_t media_loop_read = 0;
-static const constexpr uint32_t MEDIA_LOOP_MAX_READ = 4096;
 
 media_state_t media_get_state(void) {
     return media_state;
 }
 
+#define PREFETCH_SIGNAL_START 1
```

Comment:
please prefer named constants (e.g. `static constexpr uint32_t something`) over preprocessor macros

Reply:
hmm, this should rather be an `enum` I guess, but OK, will do.